apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,68 @@
-import { Component, OnInit } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Signal,
+  ViewEncapsulation,
+  effect,
+  inject,
+} from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CityStore } from '../../data-access/city.store';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card [list]="cities()" customClass="bg-light-green">
+      <ng-template let-item="item" #listItem>
+        {{ item.name }}
+        <ng-container ngProjectAs="delete-button">
```

Comment:
you are the second one I see who is doing this `ng-container ngProjectAs=` thing. Where did you read it ? 

You should simply add `delete-button` on your DOM element.

Reply:
@tomalaforge   i have tried <delete-button> directly but angular shows custom_schema error so i have fall back to this implementation